my_list.rb

```diff
@@ -0,0 +1,23 @@
+require_relative './my_enumerable'
+
+class MyList
+  include MyEnumerable
+
+  def initialize(*list)
+    @list = list
+  end
+
+  def each(&block)
+    @list.each(&block)
+  end
+end
+
```

Comment:
- [ ] We are missing some method here to make the enumerable working like the implementation of `my_each`

Reply:
Hi again, the only method required by the project description as I understood it was to implement each function. Could you explain which other functions I am to implement seeing as it was not described in the project requirements

Reply:
![results](https://user-images.githubusercontent.com/75745927/155715082-1763a11d-4a67-4afd-996c-6f470738b9b0.PNG)